### ISSUE TITLE
New Rule Suggestion: Swagger Capitalized Summery/Description

### ISSUE THREAD

**Issue by shanike** [darraghoriordan/eslint-plugin-nestjs-typed#179]
Hi! In my project, I expect `ApiProperty#Summary` and `ApiProperty#Description` to start with an upper-cased letter.

## Example:
PASSES:
```ts
  @ApiOperation({
    summary: "Clears the access-token cookie", // <<<<<<------------------------- c (upper case)
  })
  @Post("logout")
  postLogout(@Res() res: Response) {
  // ...
  }

```
FAILES:
```ts
  @ApiOperation({
    summary: "clears the access-token cookie", // <<<<<<------------------------- C (lower case)
  })
  @Post("logout")
  postLogout(@Res() res: Response) {
  // ...
  }
```

**Comment by darraghoriordan** [comment]
this could be a good first rule for someone! isUpper() is a pretty easy check
